egs2/aphasiabank/asr1/local/calc_macs_flops.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
I think this could be a general functionality for the modeling. It would be great if you can extend this into a general function in `pyscripts` and `scripts`. (Not a must)

Reply:
Yifan actually has a separate repo for this (https://github.com/pyf98/espnet_utils), but I can extend this in a new PR.

Reply:
Sounds cool! Either add dependency to Yifan's repo or adding it into scripts/pyscripts would be a great help.